salt/cdn-logs/init.sls

```diff
@@ -14,6 +14,16 @@
   file.managed:
     - source: salt://cdn-logs/config/fastly.logrotate.conf
 
+logrotate_time_hourly:
+  file.replace:
+    - name: /etc/systemd/system/timers.target.wants/logrotate.timer
+    - pattern: OnCalendar=daily
+    - repl: OnCalendar=hourly
+
 /etc/cron.hourly/logrotate:
```

Comment:
I think this state can be removed completely, since the job in question doesn't fire through cron anymore.